roles/config-docker-compose/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+docker_install: true
```

Comment:
@kenhitchcock This may sound a bit backwards. But can we change these `install` vars to false? If you look at the defaults in the roles.. they're set to false. I'd also prefer that folks who use this role are explicit about wanting to install these packages vs things getting installed by default.

Reply:
@tylerauerbeck done, I agree it makes more sense if the user sets the variable to install.